.github/workflows/prepare-patch-release.yml

```
@@ -7,44 +7,49 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-        with:
-          ref: ${{ github.ref_name }}
 
       - name: Set versions
         id: set-versions
         run: |
-          v=$(grep -Eo "[0-9]+.[0-9]+.[0-9]+" version.gradle.kts | head -1)
-          if [[ $v =~ ([0-9]+.[0-9]+).([0-9]+) ]]; then
+          prior_version=$(grep -Eo "[0-9]+.[0-9]+.[0-9]+" version.gradle.kts | head -1)
+          if [[ $prior_version =~ ([0-9]+.[0-9]+).([0-9]+) ]]; then
             major_minor="${BASH_REMATCH[1]}"
             patch="${BASH_REMATCH[2]}"
           else
-            echo "unexpected version: $v"
+            echo "unexpected version: $prior_version"
             exit 1
           fi
           echo "::set-output name=release-version::$major_minor.$((patch + 1))"
-          echo "::set-output name=prior-version::$v"
+          echo "::set-output name=prior-version::$prior_version"
```

Shouldn't that be `prior-release-version`?
```suggestion
          echo "::set-output name=prior-release-version::$prior_version"
```